### PR TITLE
Green dev/unix_test_workflow

### PR DESCRIPTION
**Pull request by jashook**

### CODE REVIEW

**Comment by jashook** [comment]
@dotnet-bot test CentOS7.1 x64 Checked Innerloop Build and Test

File: build-test.sh [hunk 1]
```diff
@@ -3,30 +3,14 @@
 initHostDistroRid()
 {
     __HostDistroRid=""
-    if [ "$__HostOS" == "Linux" ]; then
```
Comment (janvorli) [inline build-test.sh]: @jashook I am sorry I've missed this PR, but this doesn 't look right. We cannot use portable build everywhere. Alpine Linux and CentOS / RHEL 6 need to use nono-portable builds specific for those.

**Comment by jashook** [comment]
Ok thanks, I will maintain that in the pr into master #15448

File: build-test.sh (hunk 1 above)
Comment (jashook) [reply build-test.sh]: Do we publish non-portable alpine packages?

Comment (jashook) [reply build-test.sh]: I made the change as well for the CentOS scenario, our centOS packages are extremely out of date and restoring them will cause test failures.

Comment (janvorli) [reply build-test.sh]: @jashook we do publish non-portable Alpine packages. As for CentOS 6 (please note that it is only for the version 6, newer versions use portable build), the RID is in fact rhel.6. See the code you've deleted below, you'll see that. And we publish rhel.6 packages that are up to date.

**Pull request merged**